lib/api/controller/realtime.js

```diff
@@ -50,10 +45,12 @@ class RealtimeController extends NativeController {
    * @returns {Promise<Object>}
    */
   async subscribe (request) {
-    assertHasIndexAndCollection(request);
-    assertHasBody(request);
+    // assertion checks
+    this.getIndexAndCollection(request);
+    this.getBody(request);
 
-    // Only funnel protocol (and thus the embedded SDK) can use "propagate" option
+    // Only funnel protocol (and thus the embedded SDK) can use "propagate"
+    // option
```

Comment:
```suggestion
    // Only funnel protocol (and thus the embedded SDK) can use "propagate" option
```

Reply:
This line exceeds 80 chars.

Reply:
I didn't see ESLint rule about line length. I think we can do this on a case-by-case basis (at least as far as comments are concerned).

Reply:
Even if there was wrapping rules in eslint, I'm not sure we would want to enforce this, because our set of rules contain exceptions (such as message strings, or when wrapping would be more harmful to the code lisibility).

Here are our own wrapping rules: https://github.com/kuzzleio/meeting-notes/blob/master/team/code%20wrap%20rules.md